lib/app/4_view/0_constant/constant_color.dart

```diff
@@ -17,3 +17,13 @@ const Color kPrimaryBlue = Color(0xFF001E75);
 const Color kSecondaryBlue = Color(0xFF00299F);
 const Color kPrimaryGray = Color(0xFF6B6B6B);
 const Color kSecondaryGray = Color(0xFFEBEBEB);
+
+const Color kGray6B = Color(0xFF6B6B6B);
+const Color kGray7B = Color(0xFF7B7B7B);
+
+const Color kBlue00299F = Color(0xFF00299F);
+const Color kBlue233D87 = Color(0xFF233D87);
+const Color kBlue213B85 = Color(0xFF213B85);
```

Comment:
변수명이 너무 길어지면 변수를 사용하는 의미가 크게 없을 것 같아서요!
Color 의 앞의 0xFF제외하고 그다음에 나오는 숫자나 알파벳 2자리로 짓는 컨벤션은 어떠신가요?
```suggestion
const Color kBlue00 = Color(0xFF00299F);
const Color kBlue23 = Color(0xFF233D87);
const Color kBlue21 = Color(0xFF213B85);
```

이렇게요!

Reply:
네 좋은 생각인 것 같습니다!